src/components/pacbio/PacbioRunWellEdit.vue

```diff
@@ -283,27 +278,26 @@ const updateUsedAliquotSource = async (row, barcode) => {
   const tubeContent = await store.tubeContentByBarcode(barcode)
   if (tubeContent) {
     const type = tubeContent.type === 'pools' ? 'Pacbio::Pool' : 'Pacbio::Library'
-    const used_aliquot = createUsedAliquot({
-      id: row.item.id || '',
-      source_id: tubeContent.id,
-      source_type: type,
-      barcode,
-      volume: tubeContent.volume,
-      concentration: tubeContent.concentration,
-      insert_size: tubeContent.insert_size,
-      template_prep_kit_box_barcode: tubeContent.template_prep_kit_box_barcode,
-    })
-    if (used_aliquot.source_type === 'Pacbio::Library') {
-      used_aliquot.available_volume = store.getAvailableVolumeForLibraryAliquot({
-        libraryId: used_aliquot.source_id,
-        aliquotId: used_aliquot.id,
-        // Volume is 0 because this aliquot is not yet saved so it shouldnt count as used
-        volume: 0,
-      })
-      //initialize the volume to the available volume for the library aliquot
-      used_aliquot.volume = used_aliquot.available_volume
-    }
-    localUsedAliquots.value[index] = used_aliquot
+    const used_aliquot = createUsedAliquot(
+      {
+        id: row.item.id || '',
+        source_id: tubeContent.id,
+        source_type: type,
+        barcode,
+        volume: tubeContent.volume,
+        concentration: tubeContent.concentration,
+        insert_size: tubeContent.insert_size,
+        template_prep_kit_box_barcode: tubeContent.template_prep_kit_box_barcode,
+      },
+      type === 'Pacbio::Library'
```

Comment:
Similar comment to PacbioRunWell comment, feels like this logic should be above.